doc/reactions.md

```diff
@@ -8,166 +8,75 @@ Reactions allow SQL LRS to watch for patterns in submitted xAPI data and dynamic
 
```

Comment:
Since I did a full review, there's a typo on line 5. It should be "Reactions allow**s**"

Reply:
Oh I keep thinking "Reactions" was plural. I always forget "Reactions" is both the name of the whole thing (singular) and the name of each piece of data being stored (plural).